README.md

```diff
@@ -129,13 +129,26 @@ The default configuration assumes `mainnet` with only `btc` chain supported, and
 You can change the location of the configuration file with the `--conf=pathToConf` command line argument.
 
 ### Command line parameters
+Please note that NBXplorer uses cookie authentication by default. If you run your Bitcoin/Litecoin/Dash nodes using their daemon (like `bitcoind`, `litecoind` or `dashd`), they generate a new cookie every time you start them, and that should work without any extra configuration. 
+If you run the node(s) using the GUI versions, like Bitcoin\Litecoin\Dash Core Qt with the `-server` parameter while you set the rpcusername and rpcpassword in their `.conf` files, you must set those values for every crypto you are planning to support. 
+See samples below.
 
-#### From Source (.NET Core SDK required)
-The same settings as above, e.g.
-```dotnet run NBXplorer.dll --port=20300 --network=mainnet --nodeendpoint=127.0.0.1:32939```
+#### Run from source (.NET Core SDK required)
+```dotnet run NBXplorer.dll <parameters>```
```

Comment:
Use `build.sh` and `run.sh` scripts instead.

Reply:
If I try to use the run.ps1 on Windows 10 in Powershell I get this:
![image](https://user-images.githubusercontent.com/910321/64170044-642e3980-ce4f-11e9-9d57-5941c08f1d08.png)

So for me it was useful to see that I can use the `dotnet run` command too, if I don't want to set the permissions.

I can replace this line with a run.ps1/run.sh line, but I'd keep it there for people like me or people without powershell. It's good information.

We still might want to change it to `dotnet run -p .\NBXplorer\NBXplorer.csproj` to use the most up-to-date source.